openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -24728,6 +24728,282 @@ OS:SolarCollector:FlatPlate:PhotovoltaicThermal,
 
 \group Electric Load Center-Generator Specifications
 
+OS:Generator:MicroTurbine,
+      \memo MicroTurbine generators are small combustion turbines (e.g., 25kW to 500kW). The model
+      \memo calculates electrical power output, fuel use, standby and ancillary power.
+      \memo Energy recovery from exhaust air can be used to heat water.
+      \min-fields 12
+  A1, \field Handle
+       \type handle
+       \required-field
+  A2, \field Name
+      \required-field
+      \type alpha
+      \reference GeneratorNames
+      \reference MicroTurbineGeneratorNames
+      \reference ConnectionObject
```

Comment:
Is this a ConnectionObject?  Seems like this might be for the heat recovery part?